core/src/main/java/com/graphhopper/util/FetchWayGeometry.java

```diff
@@ -0,0 +1,20 @@
+package com.graphhopper.util;
+
+public enum FetchWayGeometry {
+    TOWER_ONLY, PILLAR_ONLY, BASE_AND_PILLAR, PILLAR_AND_ADJ, ALL;
+
+    public static int count(int count, FetchWayGeometry mode) {
```

Comment:
The names here could be more expressive. `int count(int count` does not tell me anything about what this method does. Apparently the method returns the total number of nodes that will be returned by `fetchWayGeometry` given the number of pillar nodes? So maybe we can at least rename the `count` parameter to `pillarNodes`? Maybe rename the method to `getFetchedNodeCount` or `countFetchedNodes`?

Reply:
I'll probably move the method into the only class where it is being used then we do not need to care much about the name for now :) (?)